lib/apple/index.js

```diff
@@ -112,8 +116,8 @@ exports.verifyPayment = function (payment, cb) {
 		return cb(null, result);
 	}
 
-
-	verify(apiUrls.production, { json: jsonData }, function (error, resultString) {
+	var url = sandbox ? apiUrls.sandbox : apiUrls.production;
```

Comment:
@ronkorving For some reason that didn't work for me in sandbox environment. It just failed after the first request to production with a different error code.